bindings/py/cpp_src/bindings/engine/py_Engine.cpp

```diff
@@ -388,7 +390,8 @@ namespace htm_ext
             .def("setMaxEnabledPhase", &htm::Network::getMaxPhase)
             .def("getMinEnabledPhase", &htm::Network::getMinPhase)
             .def("getMaxEnabledPhase", &htm::Network::getMaxPhase)
-			.def("run",                &htm::Network::run);
+            .def("setPhases",          &htm::Network::setPhases)
```

Comment:
what are Phases used for?

Reply:
Phases are uses to determine the order that regions compute their input.